action.yaml

```diff
@@ -18,18 +18,23 @@ inputs:
     descr: |
       Rclone WebDAV server port.
       Useful if you want to access the server within the cluster, for instance, from another job.
+  job_name:
+    default: "webdav"
+    descr: |
+      The name of WebDAV server.
+      Use it to generate a predictable job hostname.
```

Comment:
Is the description still actual?

Reply:
Actually yes, we use this input to set the job name [here](https://github.com/neuro-actions/webdav_server/pull/1/files#diff-fab4d7fb461bc6fbe9587f6c03fff98102b1c744145edcf2a993f2ff7cb05a0dR33).
Or, maybe I've got your question wrong

Reply:
I read it as "a string used to generate job name by concatenating with other things" but it is the whole job name actually.
Was `$[[ flow.title ]]-webdav` but now is `$[[ inputs.job_name ]]`.

Reply:
True, initially it was a buggy `$[[ flow.title ]]-webdav` 🙂
I've replaced it with the proper input in 6c8e866